packages/loader/container-utils/src/error.ts

```diff
@@ -86,6 +86,17 @@ export class UsageError extends LoggingError implements IFluidErrorBase {
     }
 }
 
+export class ClientSessionExpiredError extends LoggingError implements IFluidErrorBase {
+    readonly errorType = "clientSessionExpiredError";
+
+    constructor(
+        readonly fluidErrorCode: string,
+        readonly expiryMs: number,
+    ) {
+        super(fluidErrorCode, { clientSessionExpiryMs: expiryMs});
```

Comment:
nit: clientSessionExpiryMs -> timeoutMs to keep the size of telemetry smaller. Also, the details about client session expired is already encoded in the error type.